packages/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -984,5 +984,8 @@ export const getDocLinks = ({ kibanaBranch, buildFlavor }: GetDocLinkOptions): D
       context: `${KIBANA_DOCS}playground-context.html`,
       hiddenFields: `${KIBANA_DOCS}playground-query.html#playground-hidden-fields`,
     },
+    inference: {
+      createInferenceEndpoint: `${ELASTICSEARCH_DOCS}semantic-search-inference.html#infer-text-embedding-task`,
```

Comment:
@saikatsarkar056 we want to link to the main [Create inference API](https://www.elastic.co/guide/en/elasticsearch/reference/master/put-inference-api.html) instead

Reply:
Updated